src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java

```diff
@@ -136,19 +135,15 @@ private static void extractCopyright(final Document doc, final PoliceTicker dm)
      * Try to extract the publishing date from a script block.
      *
      * @param doc Document
-     * @param dm PoliceTicker
+     * @param dm  PoliceTicker
```

Comment:
Revert line?

Reply:
Yes I will do this. It's a formatter, normally for ts, to set the doc in columns for better reading.

Reply:
Done.